import { catchError } from "./packages/catchError"

catchError()